### DISCUSSION TITLE
Hiding colon in pager

### DISCUSSION THREAD

**Discussion by musjj** [sharkdp/bat#2547] [Q&A]
This isn't really a bat-specific question, but is there a way to hide the colon prompt in pagers like `less`?
My use case is that I'm using bat as a previewer in nnn, so the colon prompt just doesn't serve any useful purpose in that context other than creating visual noise.

**Comment by keith-hall** [comment]
I don't see any option in https://man7.org/linux/man-pages/man1/less.1.html for it unfortunately.
